.eslintrc

```diff
@@ -63,12 +53,33 @@
     "no-eval": "error",
     "dot-notation": "error",
     "no-trailing-spaces": "error",
-    "no-unused-expressions": ["error", { "allowShortCircuit": true, "allowTernary": true }],
+    "no-unused-expressions": [
+      "error",
+      {
+        "allowShortCircuit": true,
+        "allowTernary": true
+      }
+    ],
     "brace-style": "error",
-    "quotes": ["error", "single"],
+    "quotes": [
+      "error",
+      "single"
+    ],
     "radix": "error",
     "default-case": "error",
     "eqeqeq": "error",
-    "jsx-quotes": ["error", "prefer-double"]
-  }
+    "jsx-quotes": [
+      "error",
+      "prefer-double"
+    ]
+  },
+  "ignorePatterns": [
+    "src/test/*",
+    "src/tests/*",
+    "src/testData/*",
+    "src/*.test.*",
+    "src/**/*.spec.*",
+    "src/**/*.test.*",
+    "src/**/*.directive.*"
```

Comment:
The reason for this was because I saw `sitecore-jss-react-native` doing this and it made sense to keep everything the same, and `it` and `describe` were throwing linting errors. No problem reverting this!

Reply:
Yeah, I guess my vote would be to keep them linted if they were previously (can keep the exception in react native).

Reply:
Alright, done!